core/modes/modes.py

```diff
@@ -8,7 +8,6 @@
     "gdb": "a way to force gdb commands to be loaded",
```

Comment:
I think we want to delete this gdb line as well.

Reply:
Fixed, thanks